src/generated/model/queryCollectionRequest.ts

```diff
@@ -1,55 +1,52 @@
+ * Search.io offers a search and discovery service with Neuralsearch®, the world\'s first instant AI search technology. Businesses of all sizes use Search.io to build site search and discovery solutions that maximize e-commerce revenue, optimize on-site customer experience, and scale their online presence.  # Authentication <!-- ReDoc-Inject: <SecurityDefinitions> -->  # Rate limiting  The Search.io API employs a number of safeguards against bursts of incoming traffic to help maximize its stability. Users who send many requests in quick succession may see error responses that show up as HTTP status code `429`.  For the [UpsertRecord](/docs/api#operation/UpsertRecord) call, Search.io allows up to 175 write operations per second.  Treat these limits as maximums and don\'t generate unnecessary load. See [Handling limiting gracefully](/docs/api/#section/Rate-limiting/Handling-limiting-gracefully) for advice on handling `429`s.  We may reduce limits to prevent abuse.  ## Common causes and mitigations  Rate limiting can occur under a variety of conditions, but it\'s most common in these scenarios:  - Running a large volume of closely-spaced requests. Often this is part of a user\'s long-running reindex job. When engaging in these activities, you should try to control the request rate (see [Handling limiting gracefully](/docs/api/#section/Rate-limiting/Handling-limiting-gracefully)). - Running batch operations in parallel. Often [BatchUpsertRecords](/docs/api#operation/BatchUpsertRecords) is used as part of a user\'s long-running reindex job. A batch of records of size `N` passed to [BatchUpsertRecords](/docs/api#operation/BatchUpsertRecords) is counted as `N` [UpsertRecord](/docs/api#operation/UpsertRecord) calls for the purposes of rate limiting, as opposed to a single call. Treat batch API calls as a convenience rather than a way to get more write throughput.  ## Handling limiting gracefully  A basic technique for integrations to gracefully handle limiting is to watch for `429` HTTP status codes and build in a retry mechanism. The retry mechanism should follow an exponential backoff schedule to reduce request volume when necessary. We also recommend building some randomness into the backoff schedule to avoid a [thundering herd effect](https://en.wikipedia.org/wiki/Thundering_herd_problem).
  *
- * The version of the OpenAPI document: v4
+ * The version of the OpenAPI document: 4.0.0
  * Contact: support@search.io
  *
  * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
  * https://openapi-generator.tech
  * Do not edit the class manually.
  */
 
-import { RequestFile } from "./models";
-import { QueryCollectionRequestPipeline } from "./queryCollectionRequestPipeline";
-import { QueryCollectionRequestTracking } from "./queryCollectionRequestTracking";
+import { RequestFile } from './models';
+import { Null | boolean | number | string } from './null | boolean | number | string';
```

Comment:
@jonathaningram doesn't seem to be generating correctly 😕

Reply:
@jkaho I would probably:

1. Try a newer generator version just to see out of interest if it is smarter and doesn't generate it like this.
2. Failing that, I think I'd probably fix it in post scripts. a) delete the import and b) change `Null | ...` to `null | ...`.

WDYT?

Reply:
@jkaho also would try:

0. Remove `null` from the openapi file to see if it generates it correctly. Would be curious to see if it just can't handle `type` being `null` but it can handle all the other any of types.

Reply:
Updating the generator version seemed to have worked :)